src/app/pages/interest-form/interest-form.js

```diff
@@ -0,0 +1,74 @@
+import BaseView from '~/helpers/backbone/view';
+import {on, props} from '~/helpers/backbone/decorators';
+import {template} from './interest-form.hbs';
+
+const otherBoxSelector = '.other[type="checkbox"]',
+    otherTextSelector = 'input[name$="[other]"]';
+
+function bookInfoDict(label, value) {
+    if (!value) {
+        value = label;
+    }
+    return {
+        label,
+        value
+    }
```

Comment:
You could write this as:
```javascript
return {
    label,
    value || label
};
```
to fix two of the ESLint errors.  The third error appears to be that this function is never actually used anywhere, so if it's dead code I guess just remove it and everything should be fixed.